### DISCUSSION TITLE
pyart import problem

### DISCUSSION THREAD

**Discussion by srbrodzik** [ARM-DOE/pyart#1163] [Q&A] [answered]
I'm trying to run some borrowed code in python2 on a Debian 11 system that has python-arm-pyart 1.11.2-1 installed.  I normally use python3 & tried with this code but ran into too many things that needed changing.   I'm getting this string of errors when I try to import pyart

In [111]: import pyart

##
##
##     JJ Helmus and SM Collis, JORS 2016, doi: 10.5334/jors.119 
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-111-706b47c96bb9> in <module>()
----> 1 import pyart

/usr/local/lib/python2.7/dist-packages/pyart/__init__.py in <module>()
     45     # import subpackages
     46     from . import core
---> 47     from . import io
     48     from . import correct
     49     from . import graph

/usr/local/lib/python2.7/dist-packages/pyart/io/__init__.py in <module>()
     79 from .rsl import read_rsl
     80 from .mdv_radar import read_mdv
---> 81 from .sigmet import read_sigmet
     82 from .chl import read_chl
     83 from .cfradial import read_cfradial, write_cfradial

/usr/local/lib/python2.7/dist-packages/pyart/io/sigmet.py in <module>()
     28 from ..core.radar import Radar
     29 from .common import make_time_unit_str, _test_arguments, prepare_for_read
---> 30 from ._sigmetfile import SigmetFile, bin4_to_angle, bin2_to_angle
     31 from . import _sigmet_noaa_hh
     32 from ..util import mean_of_two_angles_deg

ImportError: /usr/local/lib/python2.7/dist-packages/pyart/io/_sigmetfile.so: undefined symbol: PyFPE_jbuf

**Comment by mgrover1** [comment, marked as answer]
Thanks for the question @srbrodzik - Py-ART does not support Python 2, as indicated in the changelogs (https://arm-doe.github.io/pyart/changelog.html#pyart-v1-11-0).

**Reply by srbrodzik** [reply]
I knew that but was hoping there might be an easy answer.  Thanks for getting back so quickly.

**Reply by mgrover1** [reply]
One other option would be to install a previous version of Py-ART, for example:

```bash
pip install arm-pyart==1.9.0
```

or 

```bash
pip install arm-pyart==1.10.1
```

**Reply by srbrodzik** [reply]
Thanks.  I'll give that a try.
